### DISCUSSION TITLE
v2.4.7-beta1 report

### DISCUSSION THREAD

**Discussion by kri164** [owntracks/android#1180] [Ideas & New Features]
Monitoring mode is not displayed in contact details.
preferencesLocatorDisplacement strings are not at Poeditor.com to translate

**Comment by growse** [comment]
Ah, apologies, I meant to sync poeditor before tagging. I'll do that now.

Just for clarity, when you say "displayed in contact details", which bit exactly do you mean? The pull-up contact sheet, or somewhere else?

**Comment by kri164** [comment]
Yes. I mean this pull up, where tha battery percent, distance and azimuth is.

Thank you


11. února 2022 9:16:40 SEČ, Andrew Rowson ***@***.***> napsal:

**Reply by growse** [reply]
I'm not sure we ever displayed the contact's monitoring mode, did we?

**Reply by jpmens** [reply]
I don't think we did.

**Comment by kri164** [comment]
Im sorry for misunderstanding, had thought the fix #1160 incorporated this feature, eg. Display monitoring mode on contact info panel.

Regards


11. února 2022 17:17:27 SEČ, Andrew Rowson ***@***.***> napsal:

**Reply by growse** [reply]
Ah I see. So that feature was for people to be able to record the mode in their database. The contact sheet is quite information-dense as it is, so adding in another property of the contact might be difficult to do neatly.
